tests/jobs/test_scheduler_job.py

```diff
@@ -3000,6 +3000,26 @@ def test_send_sla_callbacks_to_processor_sla_with_task_slas(self, dag_maker):
             expected_callback = SlaCallbackRequest(full_filepath=dag.fileloc, dag_id=dag.dag_id)
             self.scheduler_job.executor.callback_sink.send.assert_called_once_with(expected_callback)
 
+    @pytest.mark.parametrize(
+        "schedule",
+        [
+            None,
+            [Dataset("foo")],
```

Comment:
Worth adding an inverse of this? pass in timedelta/cron and see if it `self.scheduler_job.executor.callback_sink.send.assert_called_once_with(` --- maybe the test for that might already exist -- in which case I am ok 👍

Reply:
It does exist [here](https://github.com/apache/airflow/blob/3a7e89dbddf5fc60dbcb9582acf5e3a79d702c7c/tests/jobs/test_scheduler_job.py#L2988), but I should probably expand it to cover more cases. I'll do that in a bit here.